it/subscription_services_tos.md

```diff
@@ -1,15 +1,17 @@
 # Servizi Mozilla in abbonamento | Termini di servizio
 
-Versione 1.2, in vigore dal 10 gennaio 2024
-{: datetime="2024-01-10" }
+Versione 2.0, in vigore dal 5 febbraio 2024
+{: datetime="2024-02-05" }
 
 Mozilla Corporation offre vari servizi per migliorare la privacy e la sicurezza online, tra cui Mozilla VPN e Firefox Relay (i "Servizi").
 
-__Mozilla VPN__ è una Virtual Private Network, ossia una rete privata virtuale gestita in collaborazione con [Mullvad](https://mullvad.net). Questo Servizio nasconde il tuo traffico internet al tuo fornitore di servizi internet crittografando i dati ed eseguendo il routing attraverso le reti del nostro partner. Inoltre, sostituisce il tuo reale indirizzo IP per non renderlo visibile agli altri, come i siti web che visiti.
+__Mozilla VPN__ è una Virtual Private Network, ossia una rete privata virtuale gestita in collaborazione con [Mullvad](https://mullvad.net). Questo Servizio nasconde il tuo traffico Internet al tuo fornitore di servizi Internet crittografando i dati ed eseguendo il routing attraverso le reti del nostro partner. Inoltre, sostituisce il tuo reale indirizzo IP per non renderlo visibile agli altri, come i siti web che visiti.
```

Comment:
```suggestion
__Mozilla VPN__ è una Virtual Private Network, ossia una rete privata virtuale gestita in collaborazione con [Mullvad](https://mullvad.net). Questo Servizio nasconde il tuo traffico internet al tuo fornitore di servizi internet crittografando i dati ed eseguendo il routing attraverso le reti del nostro partner. Inoltre, sostituisce il tuo reale indirizzo IP per non renderlo visibile agli altri, come i siti web che visiti.
```

Don't know why these were changed to uppercase, but we should keep using web/internet lowercase for adjectives (vs nouns).